runtime/array_functions.h

```diff
@@ -924,14 +924,17 @@ mixed f$array_rand(const array<T> &a, int64_t num) {
   }
 
   int64_t size = a.count();
-  if (num > size) {
-    num = size;
-  }
+
   if (unlikely(num <= 0)) {
     php_warning("Parameter num of array_rand must be positive");
     return {};
   }
 
+  if (num > size) {
```

Comment:
May be make same waning as in PHP?
`Second argument has to be between 1 and the number of elements in the array`

Also, please add some tests.

Reply:
Fixed, added tests